packages/cli/test/integration/generators/relation.has-many.integration.js

```diff
@@ -0,0 +1,814 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/cli
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const path = require('path');
+const assert = require('yeoman-assert');
+const {expect, TestSandbox} = require('@loopback/testlab');
+
+const generator = path.join(__dirname, '../../../generators/relation');
+const {SANDBOX_FILES, SourceEntries} = require('../../fixtures/relation');
+const testUtils = require('../../test-utils');
+
+// Test Sandbox
+const SANDBOX_PATH = path.resolve(__dirname, '..', '.sandbox');
+const MODEL_APP_PATH = 'src/models';
+const CONTROLLER_PATH = 'src/controllers';
+const REPOSITORY_APP_PATH = 'src/repositories';
+
+const sandbox = new TestSandbox(SANDBOX_PATH);
+
+const hasManyrImportRegEx = /import \{Entity, model, property, hasMany\} from '@loopback\/repository';\n/;
```

Comment:
what is the `r` in `hasManyrImportRegEx` for?

Reply:
probably a typo

Reply:
I think it's probably a typo, this line was copied with no changes.